### ISSUE TITLE
@nrlw/react:lib --publishable omits workspace name within package.json

### ISSUE THREAD

**Issue by Hotell** [nrwl/nx#2486]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [x] I am running the latest version
- [x] I checked the documentation (nx.dev) and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository (not related to React, Angular or any dependency)

## Expected Behavior

`nx g @nrwl/react:lib hello --publishable`

creates `package.json` with proper workspace scope

-> `"name":"@my/hello"`

## Current Behavior

`nx g @nrwl/react:lib hello --publishable`

creates `package.json` without scope

-> `"name":"hello"`

### Context

**NOTE:**
- This issue is present in nx 8 as well
- @nrwl/node:lib generator handles this properly

---

- version of Nx used
```
 @nrwl/angular : Not Found
  @nrwl/cli : 9.0.0
  @nrwl/cypress : 9.0.0
  @nrwl/eslint-plugin-nx : 9.0.0
  @nrwl/express : Not Found
  @nrwl/jest : 9.0.0
  @nrwl/linter : 9.0.0
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : 9.0.0
  @nrwl/react : 9.0.0
  @nrwl/schematics : Not Found
  @nrwl/tao : 9.0.0
  @nrwl/web : 9.0.0
  @nrwl/workspace : 9.0.0
  typescript : 3.7.5
```
- 3rd-party libraries and their versions
- and most importantly - a use-case that fails

**Comment by jaysoo** [comment]
Definitely a bug. We'll patch it up for next release.

**Comment by vsavkin** [comment]
Thank you for submitting the issue. I fixed it in 9.0.2.

**Comment by Hotell** [comment]
thx @vsavkin !
